esm_runscripts/slurm.py

```diff
@@ -67,9 +68,11 @@ def calc_requirements_multi_srun(self, config):
 
 
     @staticmethod
-    def mini_calc_reqs(config, model, hostfile, start_proc, end_proc):
+    def mini_calc_reqs(self,config, model, start_proc, start_core, end_proc, end_core):
```

Comment:
`self` shouldn't be an argument to static methods.

Reply:
Not sure I understand what's wrong here. Do I extract path from self.path and give that as an argument?